beta-src/src/state/interfaces/GameStateMap.ts

```diff
@@ -0,0 +1,7 @@
+import GameStateMap from "../../types/state/GameStateMap";
```

Comment:
Nit: Rename File to GameStateMaps.ts (plural to match interface name)

Reply:
`GameStateMaps` and `GameStateMap` are different. 

`GameStateMaps` is composed of multiple `GameStateMaps`

Reply:
Sorry let me clarify. I meant to rename this file from `GameStateMap.ts` to `GameStateMaps.ts`

Reply:
Ahh I  see the issue, there is a discrepancy in the file names. will update